README.md

```diff
@@ -122,21 +127,20 @@ you need to take into consideration?
 
 * Ensure the API handles errors and edge cases properly.
 
-* Use dates in YYYY-MM-DD format for the API. There is no need of more
+* Use dates in YYYY-MM-DD format for the API. There is no need for more
   complicated date processing.
 
-* You are encouraged to modify or extend the database schema if you think a different model fits task better.
+* You are encouraged to modify or extend the database schema if you think a different model fits the task better.
 
 * If you have any questions, please don't hesitate to contact us at tech-recruitment@xeneta.com
 
-* We would like your feedback - Let us know how much time you spent on
-  the task or about any difficulties you run into.
+* We would like your feedback - Let us know how much time you spent on the task, or any difficulties you ran into.
```

Comment:
I really don't like this — it feels unfair. We're asking them to improve our process after having them do unpaid work in hopes of getting a job? I'd suggest removing it.

Reply:
I agree this is unfair, but this text is just wrong: this is not about feedback.
We want to know how long it took to finish the task and what the difficulties were because this is useful input for the technical interview after the task is submitted.